hapi-proto/src/main/proto/ResponseCode.proto

```diff
@@ -203,9 +203,14 @@ enum ResponseCodeEnum {
   INVALID_WIPE_KEY = 189;
   INVALID_FREEZE_KEY = 190;
   INVALID_SUPPLY_KEY = 191;
-  TOKEN_NAME_ALREADY_IN_USE = 192; // Token Name is already used
-  MISSING_TOKEN_NAME = 193; // Token Name is not provided
-  TOKEN_NAME_TOO_LONG = 194; // Token name is too long
-  INVALID_WIPING_AMOUNT = 195; // The provided wipe amount must not be negative, zero or bigger than the token holder balance
-  TOKEN_IS_IMMUTABlE = 196; // Token does not have Admin key set, thus update/delete transactions cannot be performed
+  INVALID_TOKEN_TRANSFER_AMOUNT = 192;
+  TOKEN_NAME_ALREADY_IN_USE = 193; // Token Name is already used
+  MISSING_TOKEN_NAME = 194; // Token Name is missing
+  TOKEN_NAME_TOO_LONG = 195;  // Token Name is too long
+  INVALID_WIPING_AMOUNT = 196;  // The amount to be wiped from the token is either bigger than the whole token balance or a negative number
+  INVALID_TOKEN_EXPIRY = 197;
+  TOKEN_HAS_EXPIRED = 198;  // The Token has expired
+  TOKEN_IS_IMMUTABlE = 199; // The Token is immutable
+  ACCOUNT_IS_TREASURY = 200;  // The chosen Account is a Treasury, therefore cannot be deleted
+  TREASURY_NOT_FOUND = 201; // Treasury was not found
```

Comment:
This is not used as well

Reply:
After you update `ResponseCode.proto`, please update `HAPI.html` -> `hapi-proto/gen-proto-docs.sh`